assets/installer/Components/Start.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+import { Button, Card, CardContent, CardHeader } from '@mui/material'
+import { useNavigate } from 'react-router-dom'
+
+
+export const StartComponent = ({nextStep}) => {
+  const navigate = useNavigate()
+
+  const handleStart = () => {
+    nextStep(0, navigate)
+  }
+
+  return(
+    <Card>
+      <CardHeader title="Start" />
+      <CardContent>
+        <Button onClick={handleStart} variant='contained'>Start</Button>
```

Comment:
Add message 'Welcome to the installer module. Follow next steps to complete the installation. Click `Start` to continue.'.